pkg/common/configs/configvalidator.go

```diff
@@ -734,7 +756,8 @@ func Validate(newConfig *SchedulerConfig) error {
 
 	// check uniqueness
 	partitionMap := make(map[string]bool)
-	for i, partition := range newConfig.Partitions {
+	for i := range newConfig.Partitions {
+		partition := newConfig.Partitions[i]
```

Comment:
It seems this change is redundant?

Reply:
Doing this to avoid lint error `G601: Implicit memory aliasing in for loop.`

Reply:
this lint error is caused by calling reference of `partition`, right? Is this lint error existent before?

Reply:
Yes it's exist from the beginning, you can try `./tools/golangci-lint run` and search error `G601: Implicit memory aliasing in for loop`.